src/features/authentication/profile/viewProfile/ViewProfile.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/require-default-props */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const ViewProfile = ({
+  image, firstName, lastName, followingLength, followerLength, bio, isUser,
+}) => (
+  <React.Fragment>
+    <div className="row profile" id="profile">
+      <div className="col-12 col-sm-12 col-md-2" />
+      <div className="col-12 col-sm-12 col-md-8">
+        <div className="row profile-info">
+          <div className="col-12 col-sm-12 col-md-4 profile-info profile-info-details">
+            <img
+              src={image}
```

Comment:
Well done @Oluwafayokemi  and @yomigeek. I love the way you guys broke down this feature into sub-features, I really like your use of components composition to modularize your UI it's awesome.
However, on line, 17 you guys set the `img` src with the image value from the user's profile. I feel you should have verified that the image property is not null so we won't display an empty image when the user is yet to update their profile photo.
A ternary operator with a default value will solve this.
A nice job guys.

Reply:
Thank you for the nice review @chukwuemekachm. I have implemented the feedback.